packages/website/public/llms/styleGuide.md

```diff
@@ -0,0 +1,417 @@
+- If you do use a beta distribution, keep in mind that there's no ({p5, p95}) format. You can use beta(alpha:number, beta:number) or beta({mean: number, stdev: number}) to create a beta distribution.
+- Write percentages as "5%" instead of "0.05". It's more readable.
+
+### Domains
+
+- Prefer using domains to throwing errors, when trying to restrict a variable. For example, don't write, "if year < 2023 then throw("Year must be 2023 or later")". Instead, write f(t: [2023, 2050]).
+- Err on the side of using domains in cases where you are unsure about the bounds of a function, instead of using if/throw or other error handling methods.
+- If you only want to set a min or max value, use a domain with Number.maxValue or -Number.maxValue as the other bound.
+- Do not use a domain with a complete range, like [-Number.maxValue, Number.maxValue]. This is redundant. Instead, just leave out the domain, like "foo(f)".
+
+```squiggle
+// Do not use this
+f(t: [-Number.maxValue, Number.maxValue]) + 1
+
+// Do this
+f(t) = t + 1
+```
+
+## Structure and Naming Conventions
+
+### Structure
+
+- Don't have more than 10 variables in scope at any one time. Feel free to use many dictionaries and blocks in order to keep things organized. For example,
+
+```squiggle
+@name("Key Inputs")
+inputs = {
+  @name("Age (years)")
+  age = 34
+
+  @name("Hourly Wage ($/hr)")
+  hourly_wage = 100
+
+  @name("Coffee Price ($/cup)")
+  coffee_price = 1
+  {age, hourly_wage, health_value, coffee_price}
+}
+```
+
+Note: You cannot use tags within dicts like the following:
+
+```squiggle
+// This is not valid. Do not do this.
+inputs = {
+  @name("Age (years)")
+  age = 34,
+
+  @name("Hourly Wage ($/hr)")
+  hourly_wage: 100,
+}
+```
+
+- At the end of the file, don't return anything. The last line of the file should be the @notebook tag.
+- You cannot start a line with a mathematical operator. For example, you cannot start a line with a + or - sign. However, you can start a line with a pipe character, `->`.
+- Prettier will be run on the file. This will change the spacing and formatting. Therefore, be conservative with formatting (long lines, no risks), and allow this to do the heavy lifting later.
+- If the file is over 50 lines, break it up with large styled blocks comments with headers. For example:
+
+```squiggle
+// ===== Inputs =====
+
+// ...
+
+// ===== Calculations =====
+```
+
+### Naming Conventions
+
+- Use snake_case for variable names.
+- All variable names must start with a lowercase letter.
+- In functions, input parameters that aren't obvious should have semantic names. For example, instead of "nb" use "net_benefit".
+
+### Dictionaries
+
+- In dictionaries, if a key name is the same as a variable name, use the variable name directly. For example, instead of {value: value}, just use {value}. If there's only one key, you can type it with a comma, like this: {value,}.
```

Comment:
> instead of {value: value}, just use {value}

I'd add a comma even in this first example, otherwise it's incorrect, or give the first example with two keys/values.